Files/Views/ModernShellPage.xaml

```diff
@@ -248,9 +248,9 @@
                 x:FieldModifier="public"
                 
                 MultiselectEnabled="{x:Bind InteractionViewModel.MultiselectEnabled, Mode=TwoWay}"
-                SelectAllInvokedCommand="{x:Bind SelectAllContentPageItemsCommand}"
-                InvertSelectionInvokedCommand="{x:Bind InvertContentPageSelctionCommand}"
-                ClearSelectionInvokedCommand="{x:Bind ClearContentPageSelectionCommand}"
+                SelectAllInvokedCommand="{x:Bind SelectAllContentPageItemsCommand, Mode=OneWay}"
```

Comment:
Curious: is this change necessary given the other changes e.g. `new RelayCommand(() => SlimContentPage?.SelectAllItems());`?

Reply:
I think the default is set to `OneTime`.

Reply:
I've also noticed these were set somewhere else, I've added `Mode=OneWay` because it was like that before Interaction removal and to unify with rest of those. @gave92

Reply:
Thanks @d2dyno1 that's fine by me.
@yaichenbaum I asked because give the changes to ModernShellPage.xaml.cs I think OneTime should be ok here.

Reply:
@gave92 Not so sure about `OneTime` here because it'd be set to null on home page and then it wouldn't update upon navigation

Reply:
I'd bet that when you do this, the "WorkingDirectory" value is evaluted each time the command is called, so it would update on navigation, but I'm actually not sure :)
```cs
public ICommand Cmd = new RelayCommand(() => NavigationHelpers.OpenDirectoryInTerminal(this.FilesystemViewModel.WorkingDirectory, this));
```